internal/cmd/iam/command_rbac_role_binding.go

```diff
@@ -15,6 +15,7 @@ import (
 	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
 	"github.com/confluentinc/cli/internal/pkg/errors"
 	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/resource"
```

Comment:
When there's a naming conflict, we prefix internal packages with "p". I wish there was a better solution...
```suggestion
	presource "github.com/confluentinc/cli/internal/pkg/resource"
```